### PR TITLE
Fix FF and IE tabbing/blur issues.

### LINKED ISSUE #1164: Hitting Tab won't jump to next field (regression in 0.12.3)
Hi,

the behaviour on KEY_TAB is broken in the version 0.12.3.

Here examples to reproduce it:

selectize 0.12.2 (works)
https://jsfiddle.net/sturchyn/b961qfjk/

selectize 0.12.3 (doesn't work)
https://jsfiddle.net/sturchyn/vyggj6oz/

(Given a Selectize with option `selectOnTab:true`)

Steps to reproduce:
1. Set focus on the select box.
2. Select the option 'B'. 
3. Press the key 'TAB'.

Expected result:
- The option 'B' is selected.
- The select box is closed.
- The cursor jumps to the next input field.

Actual result (with 0.12.3):
- The option 'B' is selected.
- The select is open.
- The focus is on the select box.

Many thanks for Selectize :-)

Best regards
Stanislav

### PR DESCRIPTION
**Pull request by topaxi**
Fixes #1164

### CODE REVIEW

File: src/selectize.js [hunk 1]
```diff
@@ -1772,9 +1772,13 @@ $.extend(Selectize.prototype, {
 
 		if (self.settings.mode === 'single' && self.items.length) {
 			self.hideInput();
-			setTimeout(function() {
+
+			// Do not trigger blur while inside a blur event,
+			// this fixes some weird tabbing behavior in FF and IE.
+			// See #1164
+			if (self.ignoreFocus) {
```
Comment (topaxi) [inline src/selectize.js:1779]: I've used `ignoreFocus` as it's set in the `onBlur` method, maybe I should use a different, new, boolean flag?

**Comment by ChurnM4c** [comment]
Will these changes be available soon? We could use them too in our current project. 

Cheers

File: src/selectize.js (hunk 1 above)
Comment (joallard) [reply src/selectize.js:1779]: I'm not actively developing the project, I'll have to take your word for it

**Comment by joallard** [comment]
If you could both validate whether the fix is suitable and ready, I'd merge.

**Comment by denisfl** [comment]
It's seems that this fix is OK. Let's merge :) 🚀

**Comment by nickolay** [comment]
This broke the `should close and blur dropdown after selection made if closeAfterSelect: true and in single mode` test added in #1127 (the PR that introduced the regression being fixed here). Anyone knows what's up with it or should it just be skipped for now?

**Comment by joallard** [comment]
@topaxi I've changed the way this operates in 8c76832 while keeping another test case green. As there is no test for this issue, it will be easy to break. Are you able to confirm at least that the issue has stayed fixed?

**Pull request merged**